### ISSUE TITLE
Experimental Set up

### ISSUE THREAD

**Issue by msakarvadia** [msakarvadia/topology_aware_learning#72]
- [x] try and get all of the datasets using the same optimizer/learning rate
- [x] More SoftMax values [1,2,4,6,8,10,25,50,75,100]
- [x] Scale # of nodes
- [ ] ER and SB topologies

<img width="344" alt="Image" src="https://github.com/user-attachments/assets/138b9d92-39f3-40b4-bfb3-09d78746bfb3" />

 <img width="342" alt="Image" src="https://github.com/user-attachments/assets/c32eba03-50a7-4237-8359-de868173960f" />
